test/metric_value_benchmark/net_test.go

```diff
@@ -53,16 +59,34 @@ func (m *NetTestRunner) validateNetMetric(metricName string) status.TestResult {
 		Status: status.FAILED,
 	}
 
-	dims, failed := m.DimensionFactory.GetDimensions([]dimension.Instruction{
-		{
-			Key:   "interface",
-			Value: dimension.ExpectedDimensionValue{aws.String("docker0")},
-		},
-		{
-			Key:   "InstanceId",
-			Value: dimension.UnknownDimensionValue(),
-		},
-	})
+	var (
+		dims   []types.Dimension
+		failed []dimension.Instruction
+	)
+	log.Println(fmt.Sprintf("OS VERSION IS %s", m.env.OS))
+	if strings.Contains(m.env.OS, "sles") {
+		dims, failed = m.DimensionFactory.GetDimensions([]dimension.Instruction{
+			{
+				Key:   "interface",
+				Value: dimension.ExpectedDimensionValue{aws.String("eth0")},
+			},
+			{
+				Key:   "InstanceId",
+				Value: dimension.UnknownDimensionValue(),
+			},
+		})
+	} else {
+		dims, failed = m.DimensionFactory.GetDimensions([]dimension.Instruction{
+			{
+				Key:   "interface",
+				Value: dimension.ExpectedDimensionValue{aws.String("docker0")},
+			},
+			{
+				Key:   "InstanceId",
+				Value: dimension.UnknownDimensionValue(),
+			},
+		})
+	}
```

Comment:
Frankly we could consolidate this since the only thing that differs is the name of the expected dimension. 
```go
var dimName string
if strings.HasPrefix(m.env.OS, "sles") {
  dimName = "eth0"
} else {
  dimName = "docker0"
}
dims, failed = m.DimensionFactory.GetDimensions([]dimension.Instruction{
	{
		Key:   "interface",
		Value: dimension.ExpectedDimensionValue{&dimName},
	},
	{
		Key:   "InstanceId",
		Value: dimension.UnknownDimensionValue(),
	},
})